Firmware/eeprom.cpp

```diff
@@ -71,8 +73,9 @@ void eeprom_init()
         }
         if(is_uninitialized)
         {
-            eeprom_write(&(EEPROM_Sheets_base->s[i].name[0]), static_cast<char>(i + '1'));
-            eeprom_write(&(EEPROM_Sheets_base->s[i].name[1]), '\0');
+            for (uint_least8_t a = 0; a < sizeof(Sheet::name); ++a){
+                eeprom_write(&(EEPROM_Sheets_base->s[i].name[a]), defaultSheetNames[i][a]);
```

Comment:
after moving defaultSheetNames into PROGMEM be careful to force the compiler to use load from progmem instructions